src/fetch.js

```diff
@@ -38,7 +38,10 @@ module.exports = async ({
  */
 const clean = item => {
   forEach(item, (value, key) => {
-    if (startsWith(key, `__`)) {
+    if (endsWith(key, `component`)) {
+      item.component = item[key]
```

Comment:
Using `component` as field name could potentially conflict with other keys. I suggest that. you use `dynamicContentType` instead, just to be explicit.
```suggestion
      item.dynamicContentType = item[key]
```